cmake/scripts/run_test.cmake

```diff
@@ -53,7 +53,8 @@ IF("${EXPECT}" STREQUAL "")
 ENDIF()
 
 EXECUTE_PROCESS(COMMAND ${CMAKE_COMMAND}
-  --build ${BINARY_DIR} --target ${TRGT}
+  --build . --target ${TRGT}
```

Comment:
No, this is more complicated. I am changing
```--build directory``` without a working directory to
```--build . WORKING_DIRECTORY directory```. This should not change anything on linux and no path changed. It just turns out that cmake will call msbuild on windows which will be confused if you are in a different directory.